Runtime/GameLiftClientSettings.cs

```diff
@@ -11,21 +17,37 @@ public sealed class GameLiftClientSettings : ScriptableObject
         public string AwsRegion;
         public string UserPoolClientId;
         public string ApiGatewayUrl;
-        public string LocalUrl = "http://localhost";
-        public ushort LocalPort = 8080;
-        public bool IsLocalTest;
+        
+        public string ComputeName;
+        public string FleetID;
+        public string FleetLocation;
+        public string AuthToken;
+        public string ProfileName;
+        public bool IsAnywhereTest;
 
         public GameLiftConfiguration GetConfiguration()
         {
-            string endpoint = IsLocalTest
-                ? $"{LocalUrl}:{LocalPort}"
-                : ApiGatewayUrl;
-            string awsRegion = IsLocalTest ? "eu-west-1" : AwsRegion;
             return new GameLiftConfiguration
             {
-                ApiGatewayEndpoint = endpoint,
-                AwsRegion = awsRegion,
+                ApiGatewayEndpoint = ApiGatewayUrl,
+                AwsRegion = AwsRegion,
                 UserPoolClientId = UserPoolClientId,
+                IsGameLiftAnywhere = IsAnywhereTest,
+            };
+        }
+
+        public ServerParameters GetStartupParameters()
+        {
+            var webSocketUrl = $"wss://{AwsRegion}.api.amazongamelift.com";
+            var processId = $"fleet-{Math.Floor(Random.value * 100000)}";
```

Comment:
Use uuid instead. Don't prepend with `fleet-`, or it could be confused as fleet id. Use `process-`